### PR TITLE
[service.subtitles.tusubtitulo] v1.2.0

### PR DESCRIPTION
**Pull request by josecurioso**
### Description
Subtitles provider for tusubtitulo.com, the new site that has substituted subtitulos.es.
The old addon for subtitulos.es and tusubtitulo.com is still on the official repository but not maintained anymore since the site went down. This is practically the same code but adapted for the new site, layout and policies.
(sorry ofr any misspellings, english is not my main language)
EDIT: made this new pull request and discarded the previous one because I realised I had some mistakes with the addon.xml (not following the new guidelines). Hopefully it's ok now.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by josecurioso** [comment]
Hello? is anybody there?...

File: service.subtitles.tusubtitulo/addon.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.beautifulsoup4" version="4.5.3"/>
+  </requires>
+  <extension point="xbmc.subtitle.module" library="service.py" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">TuSubtitulo.com</summary>
+    <description lang="en_GB">Search and download subtitles from TuSubtitulo.com</description>
+    <disclaimer lang="en_GB"></disclaimer>
+    <language>en</language>
```
Comment (enen92) [inline service.subtitles.tusubtitulo/addon.xml]: language is for content (video/audio) only

**Comment by enen92** [comment]
@josecurioso thanks for contributing and sorry for the delay. New additions can take a bit of time to review. 

There is an issue though as this addon is also in the gotham branch. Can you please create a separate pull request target at the gotham branch to set the old version as broken (broken tag in addon xml)? That PR has to be merged before this one. Also please bump the addon version of this PR to a higher value than 1.0.0 (so people with the gotham version installed can receive the update as well)

**Comment by josecurioso** [comment]
Hi, no worries for the delay, I'm sure it is not htat easy.
I just made a pull request in the gotham branch tu mark both subtituloses and tusubtitulo as broken.
Since the addon has no video/audio should I just remove the language tag?
Thanks

**Comment by MartijnKaijser** [comment]
since you need to bump the gotham version you should increase the version for this one above that.
Language tag can also mean the language of subtitles available

**Comment by josecurioso** [comment]
Done, now the old one is marked as broken and in v1.1.0 and this one is in v1.2.0 and the language tag now reflects all possible subtitle languages.

**Pull request merged**